macrel/main.py

```diff
@@ -111,8 +111,9 @@ def validate_args(args):
     if not args.output and not args.output_file:
         error_exit(args, "Either --output or --file-output argument must be used")
 
-    if args.keep_fasta_headers and args.command != 'get-smorfs':
-        error_exit(args, '--keep-fasta-headers is only available for get-smorfs command')
+    # This makes the args.keep_fasta_headers be used in more modes
```

Comment:
This comment does not make sense outside of the context of this history, so it's not helpful for people in the future.